sormas-api/src/main/java/de/symeda/sormas/api/person/PersonFacade.java

```diff
@@ -53,6 +53,8 @@
 
 	List<String> getAllUuids();
 
+	Long getPersonIdByUuid(String uuid);
```

Comment:
Ids shouldn't be exposed to the API because they're an internal property; please use the UUID instead